packages/p2p/src/components/buy-sell/buy-sell.jsx

```diff
@@ -79,6 +82,11 @@ const BuySell = () => {
                 setShouldShowPopup={buy_sell_store.setShouldShowPopup}
                 table_type={buy_sell_store.table_type}
             />
+            <RateChangeModal
+                local_currency={local_currency}
+                should_show_rate_change_modal={buy_sell_store.show_rate_changed_popup}
```

Comment:
Question: do `show_rate_change_modal` and `show_rate_changed_popup` refer to the same thing? If they do, is it possible to give them the same consistent name throughout the code?

Reply:
ok. I will rename the variable